config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'restaurants#index'
   resources :restaurants
+  root 'restaurants#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.